addon/templates/components/power-select-infinity/options.hbs

```diff
@@ -0,0 +1,17 @@
+{{#vertical-collection
+    items=options
+    estimateHeight=28
+    staticHeight=true
+    bufferSize=5
```

Comment:
@bakerac4  can `estimateHeight` and `bufferSize` be set in the parent component and passed through, in case the consuming app wants to customize them (e.g. if they use custom option components and/or custom styles make it such that the option item heights are not 28px)?